client.go

```diff
@@ -1117,7 +1117,7 @@ func (a *arkClient) refreshVtxoDb(spendableVtxos, spentVtxos []types.Vtxo) error
 }
 
 func (a *arkClient) listenForBoardingTxs(ctx context.Context) {
-	ticker := time.NewTicker(2 * time.Second)
+	ticker := time.NewTicker(time.Minute)
```

Comment:
if we do a call every 10s, we would do 6 calls per minutes, 360 calls per hour, 8640 calls per day, 259200 calls per month. The limit is 500k per month so we should be ok wihtout making the detection of boarding utxo too much slow.